runtime/src/test_utils.rs

```diff
@@ -83,8 +83,7 @@ lazy_static! {
     ]
     .into_iter()
     .collect();
-    pub static ref CALLER_TYPES_SIGNABLE: Vec<Cid> =
-        vec![*ACCOUNT_ACTOR_CODE_ID, *MULTISIG_ACTOR_CODE_ID];
+    pub static ref CALLER_TYPES_SIGNABLE: Vec<Type> = vec![Type::Account, Type::Multisig];
```

Comment:
This duplicates the value already in `runtime/src/builtin/shared.js`. Don't copy it here, just change everywhere to use the existing one.